ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/KeyValueExtractor.java

```diff
@@ -69,6 +71,28 @@ public static GenericRow extractValues(final JsonObject values, final LogicalSch
     return GenericRow.fromList(vals);
   }
 
+  static JsonObject convertColumnNameCase(final JsonObject jsonObjectWithCaseInsensitiveFields) {
```

Comment:
Instead of iterating through the entries again in here and having an extra method, it would be simpler to call ServerUtils.getIdentifierText in the existing extractValues method.